model-optimizer/mo/graph/graph.py

```diff
@@ -1032,21 +1032,24 @@ def dict_includes(big: dict, sub_dict: dict, skip_attr_names=[]):
     )
 
 
-def add_opoutput(graph: Graph, node_name: str, port: int, cut: bool = True):
+def add_opoutput(graph: Graph, node_name: str, port: int, cut: bool = True, keep_output_port=None):
```

Comment:
keep_output_port=None -> keep_output_port: bool = False

Reply:
I used `None` value to not change logic - this attribute either exist and has value True or does not exist at all. So I tried to minimize impact for existing code.

PR with this attribute implementation: https://github.com/openvinotoolkit/openvino/pull/5136

Reply:
After your change 'keep_output_port' attribute with `None` value always exists in `Result` node. And the logic for handling such case with `None` value probably exists. In PR#5136 I observe `False` value. Please check if these two cases (`False` and `None`) can be emerged. I dislike that we may have three cases (true, false, and none) that makes solution not transparent. Please align the solution.

Reply:
I vote for the default value `False` instead of `None`

Reply:
Ok, default value changed.